lighthouse-core/config/default-config.js

```diff
@@ -31,6 +31,12 @@ const UIStrings = {
   diagnosticsGroupTitle: 'Diagnostics',
   /** Description of the diagnostics section of the Performance category. Within this section are audits with non-imperative titles that provide more detail on the page's page load performance characteristics. Whereas the 'Opportunities' suggest an action along with expected time savings, diagnostics do not. Within this section, the user may read the details and deduce additional actions they could take. */
   diagnosticsGroupDescription: 'More information about the performance of your application.',
+  /** Title of the Accessibility category of audits. Accessibility contains audits focused on making web content accessible to users with disabilities. Also used as a label of a score gauge; try to limit to 20 characters. */
+  a11yCategoryTitle: 'Accessibility',
+  /** Description of the Accessibility category. This is displayed immediately under the Accessibility title element. No character length limits. 'improve the accessibility of your web app' becomes link text to additional documentation. */
```

Comment:
I think you want to repeat the "This section contains audits focused on making..." description line for each of these, since (we're told) the translators don't generally cross reference the strings, so need the context for each.

The "under the Accessibility title element" may also be too specific, and may want to instead mention it's "displayed at the top of a list of audits focused on making..."